iree/compiler/Dialect/HAL/Target/LLVM/LLVMAOTTarget.cpp

```diff
@@ -81,7 +82,15 @@ class LLVMAOTTargetBackend final : public TargetBackend {
   }
 
   void buildTranslationPassPipeline(OpPassManager &passManager) override {
-    buildLLVMTransformPassPipeline(passManager);
+    auto codeGenOptions = getLLVMCodegenOptionsFromClOptions();
+    // Set target specific options.
+    // TODO(ataei): This is temporary here, should move when target specific
+    // overrides options grows.
+    llvm::Triple triple(options_.targetTriple);
+    if (triple.isWasm()) {
+      codeGenOptions.unfuseFMAOps = true;
```

Comment:
Please add a brief comment here with more context for why Wasm targets should use this option.

```suggestion
    if (triple.isWasm()) {
      // WebAssembly does not (yet) support FMA ops natively, so unfuse them.
      codeGenOptions.unfuseFMAOps = true;
```